packages/babel-parser/test/fixtures/experimental/class-private-methods/asi-failure-generator/options.json

```
@@ -1,4 +1,7 @@
 {
-  "throws": "Unexpected token (3:3)",
-  "plugins": ["classProperties", "classPrivateMethods"]
-}
+  "throws": "This experimental syntax requires enabling the parser plugin: 'privateIn' (3:3)",
```

Can we throw this error only if the token after `#foo` is actually `in`?